src/index.test.ts

```diff
@@ -5,11 +5,13 @@ it('should transform message attributes', () => {
     string: 'hello',
     array: ['hello', 'world'],
     number: 1,
+    bool: true,
   });
 
   expect(result).toEqual({
     array: {DataType: 'String.Array', StringValue: '["hello","world"]'},
     number: {DataType: 'Number', StringValue: '1'},
     string: {DataType: 'String', StringValue: 'hello'},
+    bool: {DataType: 'Boolean', StringValue: 'true'},
```

Comment:
https://docs.aws.amazon.com/sns/latest/dg/sns-message-attributes.html#SNSMessageAttributes.DataTypes

Boolean is not supported

It must be string